### PR TITLE
Fixed ISS and generalized curl for it

### PR DESCRIPTION
**Pull request by paulhandy**

### CODE REVIEW

File: bindings/src/sign/mod.rs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pub mod simple;
+//pub mod pair;
```
Comment (th0br0) [inline bindings/src/sign/mod.rs]: remove?

File: trytes/src/num.rs [hunk 2]
```diff
@@ -38,6 +38,25 @@ pub fn int2trits(v: isize, size: u8) -> Vec<Trit> {
     ret
 }
 
+pub fn round_third(input: usize) -> usize {
```
Comment (th0br0) [inline trytes/src/num.rs]: document?

File: trytes/src/num.rs [hunk 3]
```diff
@@ -38,6 +38,25 @@ pub fn int2trits(v: isize, size: u8) -> Vec<Trit> {
...
+}
+
+pub fn min_trits(v: isize) -> u8 {
+    let mut num = 0;
+    let v_abs = v.wrapping_abs();
+    while {
+        num += 1;
+        v_abs > 3isize.pow(num)
+    }
+    {}
```
Comment (th0br0) [inline trytes/src/num.rs]: delete

File: trytes/src/num.rs [hunk 4]
```diff
@@ -38,6 +38,25 @@ pub fn int2trits(v: isize, size: u8) -> Vec<Trit> {
...
+pub fn round_third(input: usize) -> usize {
+    let rem = input % TRITS_PER_TRYTE;
+    if rem == 0 {
+        input
+    } else {
+        input + TRITS_PER_TRYTE - rem
+    }
+}
+
+pub fn min_trits(v: isize) -> u8 {
```
Comment (th0br0) [inline trytes/src/num.rs]: add documentation?

**Review by th0br0 — CHANGES_REQUESTED** [review]
looking good! add some more documentation?

**Comment by th0br0** [comment]
oh, and squash!

File: trytes/src/num.rs (hunk 3 above)
Comment (paulhandy) [reply trytes/src/num.rs]: delete what? the do while loop?

**Review by th0br0 — APPROVED** [review]

**Pull request merged**